tests/test_coordinates.py

```diff
@@ -524,6 +525,26 @@ def test_roundtrip_GEO_GEI(self):
         np.testing.assert_allclose(got.data, expected.data)
         np.testing.assert_equal(got.dtype, 'GEO')
 
+    def test_roundtrip_loop(self):
+        """Roundtrip should yield input as answer (loop over systems)"""
+        tt = Ticktock(['2002-02-02T12:00:00'], 'ISO')
+        irbframes = list(spc.SYSAXES_TYPES.keys())
+        # exclude frames either not supported by IRBEM or that aren't cartesian
+        for key in ['ECIMOD', 'GDZ', 'RLL', 'CDMAG', 'SPH']:
+            irbframes.pop(irbframes.index(key))
+        for insys in irbframes:
+            for outsys in irbframes:
+                expected = spc.Coords([1, 2, 4], insys, 'car',
+                                      use_irbem=True, ticks=tt)
+                stage1 = expected.convert(outsys, 'car')
+                got = stage1.convert(insys, 'car')
+                np.testing.assert_allclose(got.data, expected.data, atol=0.001)
+                if insys in spc.SYS_EQUIV:
+                    options = [insys, spc.SYS_EQUIV[insys]]
+                    assert insys in options
```

Comment:
It's not clear what this test is doing. I can see not doing the assert_equal is the input is in our equivalent list (i.e. test just the base system). But wouldn't the assert always pass? If it's meant to be part of the test, using assertIn might give a more useful failure message...but I'm not sure what this means, so grasping at straws here.

Reply:
Typo... it should've been `assert got.dtype in options`

The context was the `if`/`else` block. It was supposed to be testing the system name was correct after the roundtrip. Because there are some "equivalent" names (i.e., GEI and TOD in IRBEM are both ECITOD in SpacePy backend; IRBEM's SPH is just spherical GEO and so we use GEO (and SPH doesn't exist in ctrans)) the options list and the assert was supposed to check that the ouput system name was either the input name or the system equivalent name.

Reply:
Any problem with `self.assertIn(got.dtype, options)` then? Tends to give a better error message on failure.